lib/nylas/contact_group.rb

```diff
@@ -4,7 +4,15 @@ module Nylas
   # Structure to represent the Contact Group schema
   # @see https://docs.nylas.com/reference#contactsid
   class ContactGroup
-    include Model::Attributable
+    include Model
+    self.resources_path = "/contacts/groups"
+    self.creatable = false
+    self.listable = true
+    self.showable = false
+    self.filterable = false
+    self.updatable = false
+    self.destroyable = false
```

Comment:
What do you think about sorting this list either alphabetically or by which are `true` then `false`?

Reply:
taken care here 1da3b949e049ad20a30dbd7287624e3084328207